package.json

```diff
@@ -7,9 +7,11 @@
   "private": true,
   "dependencies": {
     "@types/express": "^4.17.8",
+    "@types/minio": "^7.0.6",
```

Comment:
Duplicate?

Reply:
`@types/*` contains the typescript typings for the module. So this is not a duplicate